### ISSUE TITLE
Re-set the workspace name, use PWD rather than TEST_SRCDIR as appropriate or include workspace name in path [phab:D830]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1132]
_Created by **shahms** at 2016-04-13 20:19:34_: 

This change also increases the required Bazel versio to 0.2.1 as that is when the
workspace-name path changes were incorporated.  Basically, we should be limiting the
number of directory changes as a lot of Bazel is built around sticking you in the appropriate
root and passing relative paths. In particular, for tests, PWD = TEST_SRCDIR/{workspace_name}.

**Comment by kythe** [comment]
_**shahms** wrote at 2016-04-13 20:38:11_:

  - Update new test to use PWD rather than TEST_SRCDIR

**Comment by kythe** [comment]
_**zarko** wrote at 2016-04-13 23:05:11_:

This fails a couple of tests on OS X. Here's the log from `kwazthis_test`:

```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
-----------------------------------------------------------------------------
2016/04/13 23:02:07 Starting serving pipeline
2016/04/13 23:02:07 Writing partial edges
2016/04/13 23:02:07 Writing complete edges
2016/04/13 23:02:07 Writing EdgeSets
2016/04/13 23:02:07 Writing decoration fragments
2016/04/13 23:02:07 Writing completed FileDecorations
2016/04/13 23:02:07 Writing CrossReferences
2016/04/13 23:02:08 search.Search:	471.206µs
2016/04/13 23:02:08 Could not locate file {path:"kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/Generics.java" }
```

**Comment by kythe** [comment]
_**shahms** wrote at 2016-04-14 15:25:01_:

Did you run `bazel clean` first? Changing the workspace name generally requires that.

**Comment by kythe** [comment]
_**shahms** wrote at 2016-04-14 21:52:33_:

PTAL? Landed the fix for OS X and rebased.

**Comment by kythe** [comment]
_**shahms** wrote at 2016-04-14 21:53:05_:

  - Update new test to use PWD rather than TEST_SRCDIR and rebase

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2016-04-14 21:58:01_

**Comment by kythe** [comment]
_**shahms** closed via commit f2d34d071b1971437e7dc8ac91f0068ed5f952d2 at 2016-04-14 22:05:20_
